src/node.js

```diff
@@ -28,9 +33,30 @@ function sendInfraMetrics () {
   analyticsClient.trackInfraMetrics(util.getTotalOrbitStores(), util.getOrbitDBDiskUsage, util.getIPFSDiskUsage())
 }
 
+function prepareIPFSConfig () {
+  if (AWS_BUCKET_NAME) {
+    if (!IPFS_PATH || !AWS_BUCKET_NAME || !AWS_ACCESS_KEY_ID || !AWS_SECRET_ACCESS_KEY) {
+      throw new Error('Invalid IPFS + s3 configuration')
+    }
+
+    const repo = ipfsRepo({
+      path: IPFS_PATH,
+      bucket: AWS_BUCKET_NAME,
+      accessKeyId: AWS_ACCESS_KEY_ID,
+      secretAccessKey: AWS_SECRET_ACCESS_KEY
+    })
+    return { repo }
+  } else if (IPFS_PATH) {
+    return { repo: IPFS_PATH }
+  }
+
+  return {}
+}
+
```

Comment:
great, nice having this configured to fallback to to running without s3 if not given s3 params